Source/Routing and Faulting/Help/HTML Help Source/Main.htm

```diff
@@ -21,6 +21,9 @@
 			<hr width="100%" noshade>
 		</noscript>
 		<h1>Routing and Faulting</h1>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tristique et egestas quis ipsum suspendisse ultrices gravida dictum fusce. Laoreet non curabitur gravida arcu ac tortor dignissim. Enim praesent elementum facilisis leo. Viverra mauris in aliquam sem. Tristique et egestas quis ipsum. Et egestas quis ipsum suspendisse ultrices gravida. Imperdiet nulla malesuada pellentesque elit eget gravida cum sociis natoque. Vitae congue eu consequat ac felis donec. Dictum sit amet justo donec enim diam vulputate. Semper auctor neque vitae tempus quam. Quis eleifend quam adipiscing vitae proin. Mollis nunc sed id semper. Lorem ipsum dolor sit amet consectetur adipiscing elit. Risus ultricies tristique nulla aliquet enim tortor at.</br>
+		<p class="Body">The <strong>Routing and Faulting Custom Device</strong> is a VeriStand add-on that configures aliases and routing channels.</p>
+		<h2>What do you want to do?</h2>
+		<h3>Script Routing and Faulting Configuration</h3>
+		<p class="Body">Open the scripting examples on disk from <strong>Source/Scripting Examples/Scripting Examples.lvproj</strong>. Run <strong>Configure Routing and Faulting Custom Device.vi</strong> to add routing channels, states, and connections.</p>
```

Comment:
> `<strong>Source/Scripting Examples/Scripting Examples.lvproj</strong>`

Should this be the installed location (TBD)?

Reply:
Let's remove the folder location and just mention the example name. Once this is in the example database people should able to find it quickly.